External/FEXCore/Source/Interface/Core/OpcodeDispatcher/X87.cpp

```diff
@@ -782,6 +782,12 @@ void OpDispatchBuilder::X87UnaryOp(OpcodeArgs) {
   // Overwrite the op
   result.first->Header.Op = IROp;
 
+  if constexpr (IROp == IR::OP_F80SIN ||
```

Comment:
Perhaps line the ops up like so:

```cpp
   if constexpr (IROp == IR::OP_F80SIN ||
                 IROp == IR::OP_F80COS) {
```

which makes it a little nicer to vertically scan down the ops being handled

Reply:
Aligned.